ui/app/src/components/PreviewDialog/PreviewDialogContainer.tsx

```diff
@@ -52,11 +53,12 @@ export function PreviewDialogContainer(props: PreviewDialogContainerProps) {
           <AsyncVideoPlayer playerOptions={{ src: url, autoplay: true, ...(mimeType ? { type: mimeType } : {}) }} />
         );
       case 'page':
+        const pagePreviewUrl = `${guestBase}${getPreviewURLFromPath(url)}?crafterCMSGuestDisabled=true`;
         return (
           <>
             {isLoading && <LoadingState />}
             <IFrame
-              url={url}
+              url={pagePreviewUrl}
```

Comment:
Not sure about this. How does this fix things?

Reply:
What is now pagePreviewUrl was sent by the search hook as url, but it is used in this component for multiple things, and one of the usages is to retrieve a detailedItem, so that is the reason why there was an incorrect call (wrong path). Now the item path is the one being sent, and the previewUrl is defined in the PreviewDialog component for the iframe to render the proper page